addons/godot-xr-tools/assets/PlayerBody.gd

```diff
@@ -39,16 +39,16 @@ export var push_rigid_bodies := true
 export (Resource) var physics = null setget set_physics, get_physics
 
 ## Path to the ARVROrigin node
-export (NodePath) var origin = null
+export var origin := NodePath()
```

Comment:
I prefer `=` over `:=` personally. Also keeping in mind this will before `@export var origin : NodePath = NodePath()` in Godot 4 (though in Godot 4 it's not needed to add the `= NodePath()` when typed, the same might apply for Godot 3)
My personal preference though.

Reply:
There btw is also a new `@export_nodepath(type) var origin` notation in Godot 4 that allows you to limit what type of object you want to nodepath to point to but I think it still has some problems.

Reply:
For Godot3 it seems like the property-editor use both the typing system and export-hints; however the static-analysis typing system completely ignores export-hints. In other words:
```
export var some_int: int        # Export of explicitly typed integer, displays as int in the property-editor
export var some_int := 123      # Export of explicit (via assignment) typed integer, displays as int in the property-editor
export (int) var some_variant   # Export an untyped variant, but have it display as an int in the property-editor
```

So as far as I can tell ```export (NodePath) var origin``` would seem to be declaring an untyped variant without any code-editor hints, but shows correctly in the property-editor.

Reply:
Maybe I'm getting a little too overly zealous about strong typing for Godot3 where all it can provide is some developer hints. For Godot4 I would really like to get the performance benefits of strong typing.

For Godot3 let's stick with ```export (NodePath) var origin```

Reply:
Removed numerous NodePath exports as no-longer needed, and the remaining ones use the preferred approach.